e2e/operator_installation_test.go

```diff
@@ -0,0 +1,70 @@
+package e2e_test
+
+import (
+	. "github.com/maistra/istio-workspace/e2e/infra"
+	"github.com/maistra/istio-workspace/pkg/naming"
+	testshell "github.com/maistra/istio-workspace/test/shell"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"strings"
+	"time"
+)
+
+var _ = Describe("Operator Installation Tests", func() {
+
+	Context("local installation", func() {
+
+		It("install ike operator into specified namespace", func() {
+			// given
+			projectName := "ike-local-installation-defined-namespace-" + naming.RandName(16)
+			<-testshell.Execute(NewProjectCmd(projectName)).Done()
+			defer func() {
+				<-testshell.Execute(DeleteProjectCmd(projectName)).Done()
+			}()
+			PushOperatorImage(projectName)
+
+			// when
+			<-testshell.Execute("ike install-operator -l -n " + projectName).Done()
```

Comment:
call InstallLocalOperator(ns)?

Reply:
Sure, but isn't it hiding q bit the actual subject of the test?